atomics/T1110.001/T1110.001.yaml

```diff
@@ -81,3 +81,47 @@ atomic_tests:
         }
       }
       Write-Host "End of bruteforce"
+- name: Brute Force Credentials of single Azure AD user
+  auto_generated_guid: 5a51ef57-299e-4d62-8e11-2d440df55e69
+  description: |
+    Attempt to brute force Azure AD user via AzureAD powershell module.
+  supported_platforms:
+  - azure-ad
+  input_arguments:
+    username:
+      description: Account to bruteforce
+      type: String
+      default: bruce.wayne@contoso.com
```

Comment:
the default value is fine in this line but can you enhance the description of the dependencies which says :
" We encourage users running this atomic to add a valid microsoft account domain; for eg: bruce.wayne@<valid_ms_account.com>" 

This will be very useful for users who are running these atomics to improve their detection coverage